### PR TITLE
add gender field on hairdresser

### PR DESCRIPTION
**Pull request by nlehuby**
it will be very useful to distinguish barber shops from regular hairdressers

### CODE REVIEW

File: data/presets/shop/hairdresser.json [hunk 1]
```diff
@@ -7,6 +7,10 @@
     "terms": [
         "beard"
     ],
+    "fields": [
+        "name",
+        "gender"
+    ],
```
Comment (tyrasd) [inline data/presets/shop/hairdresser.json:13]: there's some superfluous whitespace here
```suggestion
    ],
```

File: data/presets/shop/hairdresser.json [hunk 2]
```diff
@@ -7,6 +7,10 @@
     "terms": [
         "beard"
     ],
+    "fields": [
+        "name",
```
Comment (tyrasd) [inline data/presets/shop/hairdresser.json:11]: Please also include the regular fields of the `shop` preset:

```suggestion
        "name",
        "operator",
        "address",
        "building_area_yes",
        "opening_hours",
        "payment_multi",
        "phone",
```

**Review by tyrasd — CHANGES_REQUESTED** [review]
Some minor tweaks are needed (see below).

I'm not super happy with reusing the `gender` preset for this, as the description "Gender" could be maybe a bit confusing (at least in some translations). But as I'm also not sure what a better description could be (_Haircut Style_ perhaps? or _Customers Served_??) right now, so I guess it's fine for the moment.

**Review by tyrasd — APPROVED** [review]

**Comment by nlehuby** [comment]
Thanks :+1:

**Pull request merged**